src/main/java/chess/repository/ChessRepository.java

```diff
@@ -0,0 +1,71 @@
+package chess.repository;
+
+import chess.domain.game.ChessGame;
+import chess.domain.piece.Color;
+import chess.utils.Serializer;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.jdbc.core.RowMapper;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+
+@Repository
+public class ChessRepository {
+    private final JdbcTemplate jdbcTemplate;
+
+    public ChessRepository(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
+    }
+
+    public Optional<String> findGame(String title) {
+        String findTitleQuery = "SELECT id FROM chess_game WHERE BINARY title = ?";
+        return jdbcTemplate.queryForList(findTitleQuery, String.class, title)
+                .stream()
+                .findAny();
+    }
+
+    public String addGame(ChessGame chessGame, String title) {
+        String addingGameQuery = "INSERT INTO chess_game (turn, finished, board, title) VALUES (?, ?, ?, ?)";
+        jdbcTemplate.update(addingGameQuery, chessGame.getTurn(), chessGame.isOver(), Serializer.serializeGame(chessGame)
+                , title);
+        String findingGameQuery = "SELECT MAX(id) FROM chess_game";
+        return jdbcTemplate.queryForObject(findingGameQuery, String.class);
```

Comment:
```suggestion
        return jdbcTemplate.queryForObject(findingGameQuery, Long.class);
```

`Long`을 사용하기로 결정하셨으면 여기서도 이렇게 해도 되지 않나요?

Reply:
맞습니다!